02 Algorithm Reference/04 Importing Custom Data/03 Creating and Reading Custom Data.html

```diff
@@ -84,7 +82,7 @@
     ''' Weather based rebalancing'''
 
     def GetSource(self, config, date, isLive):
-        source = "https://www.wunderground.com/history/airport/{0}/{1}/1/1/CustomHistory.html?dayend=31&monthend=12&yearend={1}&format=1".format(config.Symbol, date.year);
+        source = "https://www.dropbox.com/s/m7894ciswqgsv9v/custom_weather_data.csv?dl=1"
```

Comment:
Please change the demo URL to QC's dropbox:
https://www.dropbox.com/s/8v6z949n25hyk9o/custom_weather_data.csv?dl=1